packages/shared/src/UI/components/AssetsManagement/CollectionList.tsx

```diff
@@ -164,12 +164,33 @@ export const CollectionList = memo(function CollectionList({
     const { collections, currentCollection, currentCollectionId, setCurrentCollectionId, loading, error, retry } =
         CollectionsContext.useContainer()
 
+    const containerRef = useRef<HTMLDivElement>(null)
+    const mainColumnRef = useRef<HTMLDivElement>(null)
+    const forkedMainColumnRef = useForkRef(mainColumnRef, scrollElementRef)
+    const scrollToTop = useCallback(() => {
+        if (disableWindowScroll) {
+            mainColumnRef.current?.scrollTo({ top: 0, behavior: 'smooth' })
+        } else {
+            const rect = containerRef.current?.getBoundingClientRect()
+            if (!rect) return
+            // 53, height of the sticky bar of Twitter,
+            // 96, height of the header of web3 tab
```

Comment:
We should concern about other SNS providers.

Reply:
@guanbinrui PTAL